converter/src/main/java/gov/cms/qpp/conversion/validate/ClinicalDocumentValidator.java

```diff
@@ -49,15 +49,11 @@ protected void performValidation(final Node node) {
 
 		if (!containsError(Detail.forProblemAndNode(ProblemCode.CLINICAL_DOCUMENT_MISSING_PROGRAM_NAME.format(VALID_PROGRAM_NAMES), node))) {
 			String programName = Optional.ofNullable(node.getValue(ClinicalDocumentDecoder.PROGRAM_NAME)).orElse("<missing>");
-			String entityType = Optional.ofNullable(node.getValue(ClinicalDocumentDecoder.ENTITY_TYPE)).orElse("<missing>");
 
 			forceCheckErrors(node).valueIn(ProblemCode.CLINICAL_DOCUMENT_INCORRECT_PROGRAM_NAME.format(programName, VALID_PROGRAM_NAMES),
 				ClinicalDocumentDecoder.PROGRAM_NAME, ClinicalDocumentDecoder.MIPS_PROGRAM_NAME,
 				ClinicalDocumentDecoder.CPCPLUS_PROGRAM_NAME, ClinicalDocumentDecoder.PCF, ClinicalDocumentDecoder.APP_PROGRAM_NAME);
 
-			if (ClinicalDocumentDecoder.ENTITY_VIRTUAL_GROUP.equals(entityType)) {
-				forceCheckErrors(node).value(ProblemCode.VIRTUAL_GROUP_ID_REQUIRED, ClinicalDocumentDecoder.ENTITY_ID);
-			}
 		}
```

Comment:
Was this breaking code for PCF submissions? I believe virtual groups are still required to have their own virtual group entity id.

Reply:
Yes, if that is left in, then `PCF-1g-MIPS_VIRTUALGROUP_06162021.xml` fails.